rootex/framework/scene.cpp

```diff
@@ -249,28 +249,27 @@ bool Scene::snatchChild(Scene* child)
 	}
 
 	Unordered_set<Ptr<Scene>>& children = child->getParent()->getChildren();
-	for (auto&& child_scene : children)
+	for (auto& child_scene : children)

```

Comment:
is there a reason to use an lvalue reference instead of rvalue?

Reply:
This is the universal binder, not an rvalue reference. However, this diff doesn't seem to change anything.